en/releases/release-1.3.0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: TiDB Operator 1.3.0 Release Notes
+---
+
+# TiDB Operator 1.3.0 Release Notes
+
+Release date: February 12, 2022
+
+TiDB Operator version: 1.3.0
+
+## Compatibility Change
+
+- Due to changes in [#4400](https://github.com/pingcap/tidb-operator/pull/4400), if deploying TiDB cluster across Kubernetes clusters by TiDB Operator v1.3.0-beta.1 or earlier, upgrading TiDB Operator to v1.3.0 directly will cause failed rolling upgrading. You have to upgrade TiDB Operator by following steps:
+    1. Update CRD.
+    2. Add new `spec.acrossK8s` as `true` in TidbCluster spec.
+    3. Upgrade TiDB Operator.
+
+## New Features
+
+- Add new field `spec.tidb.tlsClient.skipInternalClientCA` to skip server certificate verification when internal components access TiDB ([#4388](https://github.com/pingcap/tidb-operator/pull/4388), [@just1900](https://github.com/just1900))
+- Support configuring DNS config for Pods of all components ([#4394](https://github.com/pingcap/tidb-operator/pull/4394), [@handlerww](https://github.com/handlerww))
+- Add new field `spec.tidb.initializer.createPassword` to support setting a random password for TiDB when deploying new cluster ([#4328](https://github.com/pingcap/tidb-operator/pull/4328), [@mikechengwei](https://github.com/mikechengwei))
+- Add new field `failover.recoverByUID` to support one-time recover for TiKV/TiFlash/DM Worker ([#4373](https://github.com/pingcap/tidb-operator/pull/4373), [@better0332](https://github.com/better0332))
+- Add new field `sepc.pd.startUpScriptVersion` to use `dig` command not `nslookup` to lookup domain in startup script of PD ([#4379](https://github.com/pingcap/tidb-operator/pull/4379), [@july2993](https://github.com/july2993))
```

Comment:
```suggestion
- Add a new field `sepc.pd.startUpScriptVersion` to use the `dig` command instead of `nslookup` to lookup domain in the startup script of PD ([#4379](https://github.com/pingcap/tidb-operator/pull/4379), [@july2993](https://github.com/july2993))
```